Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -119,7 +118,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             if ($shouldNotCreateDatabase) {
                 $output->writeln(sprintf('<info>Database <comment>%s</comment> for connection named <comment>%s</comment> already exists. Skipped.</info>', $name, $connectionName));
             } else {
-                $tmpConnection->getSchemaManager()->createDatabase($name);
+                $schemaManager = method_exists($tmpConnection, 'createSchemaManager')
+                    ? $tmpConnection->createSchemaManager()
+                    : $tmpConnection->getSchemaManager();
```

Comment:
there is another usage of `getSchemaManager` in the computation of `$shouldNotCreateDatabase`

Reply:
Ah damn missed that. Will create a follow-up PR :+1: